tests/test_docking.py

```diff
@@ -35,18 +35,18 @@ def test_pose_dock(self):
             outdir,
         )
 
-        self.assertTrue(Path("test_pd_outdir/338/").exists())
-        self.assertTrue(Path("test_pd_outdir/338/338_pd_1000_host.pdb").exists())
-        self.assertTrue(Path("test_pd_outdir/338/338_pd_1000_guest.sdf").exists())
-        self.assertTrue(Path("test_pd_outdir/43/").exists())
-        self.assertTrue(Path("test_pd_outdir/43/43_pd_1000_host.pdb").exists())
-        self.assertTrue(Path("test_pd_outdir/43/43_pd_1000_guest.sdf").exists())
+        self.assertTrue(Path(f"{outdir}/338/").exists())
+        self.assertTrue(Path(f"{outdir}/338/338_pd_1000_host.pdb").exists())
+        self.assertTrue(Path(f"{outdir}/338/338_pd_1000_guest.sdf").exists())
+        self.assertTrue(Path(f"{outdir}/43/").exists())
+        self.assertTrue(Path(f"{outdir}/43/43_pd_1000_host.pdb").exists())
+        self.assertTrue(Path(f"{outdir}/43/43_pd_1000_guest.sdf").exists())
 
-        for f in Path("test_pd_outdir").glob("*/*.*"):
+        for f in Path(outdir).glob("*/*.*"):
             Path.unlink(f)
-        for d in Path("test_pd_outdir").glob("*"):
+        for d in Path(outdir).glob("*"):
             d.rmdir()
-        Path("test_pd_outdir").rmdir()
+        Path(outdir).rmdir()
```

Comment:
if the assertion/tests fail do the directories still get cleaned up? not sure if you need a context manager or a use a context managed tempfile/tempdir?

Reply:
good question! I'll have to google that